python/cugraph/tests/dask/opg_pagerank_testing.py

```diff
@@ -24,9 +25,9 @@ def test_dask_pagerank():
     gc.collect()
     cluster = LocalCUDACluster()
     client = Client(cluster)
+    Comms.initialize()
```

Comment:
Should this be `reinitialize` like RMM? We should document this API and this should be optional when using default parameters.

Some doc on behavior :
`To configure options to be used in cuGraph before loading, simply do the above before you import cugraph. You can re-initialize the cugraph communicator with different settings at run time by calling reinitialize().`

Reply:
Depends on whether we want to reinitialize. rmm allows reinitialization. On the contrary, currently the Comms here allows initialization just once and uses that throughout. Any reinitialization at a later point might cause some assumptions to be False(rank initialization and partition order of workers) and precomputed distributed data with new comms initialization might misbehave.

Reply:
Can Comms.initialize() be embedded elsewhere or do we need to explicitly call it?

Reply:
We could embed it within but it can still be initialized only once. The point would be the user to be aware of the communicator, because an explicit destroy is needed too